lib/src/cache_entry.dart

```diff
@@ -1,14 +1,14 @@
 part of dcache;
 
 class CacheEntry<K, V> {
-  DateTime insertTime;
-  K key;
-  V value;
+  final DateTime insertTime;
+  final K key;
+  final V value;
   bool updating = false;
   DateTime lastUse;
   int use = 0;
 
-  CacheEntry(this.key, this.value, this.insertTime) {
+  CacheEntry(this.key, this.value) : insertTime = DateTime.now() {
```

Comment:
Is it possible to put it optional rather than delete it ?

Reply:
of course! Feel free.